logging/cluster-logging-uninstall.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can remove the {logging} from your {product-title} cluster by removing installed Operators and related custom resources (CRs).
+You can remove {logging} from your {product-title} cluster by removing installed Operators and related custom resources (CRs).
```

Comment:
The variables in this file are not rendering in preview.

Reply:
Fixed!